node/node_test.go

```diff
@@ -547,7 +534,8 @@ func TestSpacemeshApp_NodeService(t *testing.T) {
 	}
 
 	// This stops the app
```

Comment:
this comment is confusing after your change

Reply:
Updated